config/locales/money_navigator_tool/money_navigator_tool.en.yml

```diff
@@ -136,6 +136,7 @@ en:
             text: 'I am not self employed'
             default: true
             sort_order: 1
+            hidden: true
```

Comment:
We shouldn't hide this: it is still necessary if the user has indicated they are not self employed in an earlier question. Additionally any display logic like this appears to be outside the scope of this TP card so I would remove that from this work and deal with it separately if necessary (which I do not believe it is).

Reply:
Hey David, As I understand it this question is _only_ displayed when the user has selected that they are self-employed. The question text reads "If you are self-employed do you know your income". I am not self employed is automatically selected as the default, so it will be selected even if the user doesn't answer this, which is what happens if they select any other answer to the previous question. 
The reason this has been added into this update is that on asking for clarification of the requested changes to the logic, a complicated change was requested in the scenario that the user has told us they are self-employed in the previous question, and then told us they aren't in this question.  To avoid this unnecessary complication we should not give the users the ability to select contradictory answers.

Reply:
Yes, though that is question is excluded dynamically. If you load the tool without JS for instance you have all the questions. In this case it would be possible to answer that you are not self-employed but still be forced to answer Yes/No to the question that is only relevant to self-employed people. That is why the default is set to not self-employed. This consideration could also apply to some assistive technologies.